src/domains/nft/hooks.ts

```diff
@@ -8,6 +8,12 @@ export const useUserNfts = () =>
     queryFn: getAllUserNfts,
   });
 
+export const useCurrentNft = (id: string) =>
+  useQuery({
+    queryKey: ["nfts-current"],
+    queryFn: () => getNftById(id),
+  });
+
```

Comment:
Is there a reason to call it current nft?
Could be like this:
```suggestion
export const useNft = (id: string) =>
  useQuery({
    queryKey: ["nfts", id],
    queryFn: () => getNftById(id),
  });

```